usr/share/rear/skel/default/etc/scripts/system-setup

```diff
@@ -151,15 +152,17 @@ else
     rear_debug_options=''
 fi
 
-# Launch rear recover automatically:
+reboot_command="${RECOVERY_REBOOT_COMMAND:-reboot}"
```

Comment:
I'd make sourcing `default.conf` mandatory and set the default value there. This follows better our custom which is to set all default values in `default.conf` to make it easier to find and understand them.

Reply:
yes

Reply:
Fixed via
https://github.com/rear/rear/pull/3070/commits/961b4883f9a241d4b92882293c197ce25835fda5